zone/spells.cpp

```diff
@@ -478,7 +478,7 @@ bool Mob::DoCastSpell(uint16 spell_id, uint16 target_id, CastingSlot slot,
 	if (IsClient() && slot == CastingSlot::Item && item_slot != 0xFFFFFFFF) {
 		auto item = CastToClient()->GetInv().GetItem(item_slot);
 		if (item && item->GetItem())
-			MessageString(Chat::Spells, BEGINS_TO_GLOW, item->GetItem()->Name);
+			MessageString(Chat::FocusEffect, BEGINS_TO_GLOW, item->GetItem()->Name);
```

Comment:
Live doesn't have this message (anymore?) when casting items so it should probably be removed or rule gated. Tested with a couple random items (Oil of Fennin Ro and Elder Spiritist Boots)

Reply:
Is this the only message they removed?

Reply:
Furious bash and the Client::GetFocusEffect messages still occur. I dunno what merc focus effects are but the function is also `GetFocusEffect` so it's probably ok